frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunTitle.tsx

```diff
@@ -15,9 +15,9 @@ const PipelineRunTitle: React.FC<PipelineRunTitleProps> = ({ run }) => {
   const { icon, label } = computeRunStatus(run);
 
   return (
-    <Split hasGutter>
-      <SplitItem>{run.name}</SplitItem>
-      <SplitItem>
+    <Split>
+      <SplitItem className="pf-v5-u-text-break-word">{run.name}</SplitItem>
+      <SplitItem className="pf-v5-u-px-md">
```

Comment:
I prefer `hasGutter` over applying additional classes. 
If there's a gap missing between the title and the actions, it's because we likely are missing spacing in the header layout of `ApplicationPage`.